portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -251,27 +253,69 @@ function allowUserSubmit() {
         }
       });
 }
-google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawStateIncarcerationChart);
-
-/** Creates chart and adds it to the page. */
-function drawStateIncarcerationChart() {
-  const stateData = new google.visualization.DataTable();
-  stateData.addColumn('string', 'Race');
-  stateData.addColumn('number', 'Count');
-  stateData.addRows([
-    ['Whites', 201],
-    ['Blacks', 1767],
-    ['Hispanics', 385],
-  ]);
-
-  const stateChartDimensions = {
-    'title': 'Incarceration Rates (per 100,000) by Race in California',
-    'width': 500,
-    'height': 400,
+
+drawTimeSeriesChart();
```

Comment:
instead of just leaving this in the code, either add it to an onLoad function or export it at the top of the file.